src/lib/contributers.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+import Axios from 'axios'
+
+
+const useContributerData = props => {
+    const URL = 'https://api.github.com/repos/anushbhatia/Social-Scheduler/stats/contributors'
+    let [data, setData] = useState([]);
```

Comment:
Replace let with const since it has no effect on non-primitives and gives some safety with re assigning values